api/python/notebooks/api_demo/census_duplicated_cells.ipynb

```diff
@@ -0,0 +1,464 @@
+       "adipose tissue   True             48b37086-25f7-4ecd-be66-f5bb378e3aea     6777\n",
+       "pancreas         True             48b37086-25f7-4ecd-be66-f5bb378e3aea     6201\n",
+       "skin of body     True             48b37086-25f7-4ecd-be66-f5bb378e3aea     4454\n",
+       "large intestine  True             48b37086-25f7-4ecd-be66-f5bb378e3aea     1887\n",
+       "Name: count, dtype: int64"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "tabula_muris_obs.value_counts()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can see all cells across the tissues are labelled as `True` for `is_primary_data`.\n",
+    "\n",
+    "But what if we select cells from the dataset that only contains cells from the liver: \"Liver - A single-cell transcriptomic atlas characterizes ageing tissues in the mouse - 10x\".\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tabula_muris_liver_dataset_id = \"6202a243-b713-4e12-9ced-c387f8483dea\"\n",
+    "\n",
+    "with cellxgene_census.open_soma() as census:\n",
+    "    tabula_muris_liver_obs = census[\"census_data\"][\"mus_musculus\"].obs.read(\n",
+    "        value_filter=f\"dataset_id == '{tabula_muris_liver_dataset_id}'\", column_names=[\"tissue\", \"is_primary_data\"]\n",
+    "    )\n",
+    "\n",
+    "    tabula_muris_liver_obs = tabula_muris_liver_obs.concat().to_pandas()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And we take a look at counts for the unique combinations of values."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "tissue  is_primary_data  dataset_id                          \n",
+       "liver   False            6202a243-b713-4e12-9ced-c387f8483dea    7294\n",
+       "Name: count, dtype: int64"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "tabula_muris_liver_obs.value_counts()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can see that:\n",
+    "\n",
+    "1. This dataset only contains cells from liver.\n",
+    "2. All cells are labelled as `False` for `is_primary_data`. **This is because the cells are marked as duplicate cells of the main Tabula Muris Senis dataset.**\n",
+    "\n",
+    "##  Filtering out duplicates cells\n",
+    "\n",
+    "In some cases you may be interested in getting all cells for a specific biological context, for example *\"all natural killer cells from blood of female cells with COVID-19\"* but you need to be aware that there is a chance you end up with some duplicate cells.\n",
```

Comment:
```suggestion
    "If you are interested in getting all cells for a specific biological context, you need to be aware that you may end up with duplicate cells.\n",
```
mainly, I felt the `for example` was redundant w/below, and then also simplified the sentence.